plugins/kubernetes/app/models/kubernetes/resource.rb

```diff
@@ -179,6 +182,22 @@ def update
         end
       end
 
+      # TODO: remove name hack https://github.com/abonas/kubeclient/issues/427
+      def server_side_apply(template)
+        with_header 'application/apply-patch+yaml' do # NOTE: we send json but say +yaml since +json gives a 415
```

Comment:
lolwut? any upstream fixes?

Reply:
we checked kubernetes source code and this is hardcoded 🤷‍♂